modules/gdscript/tests/scripts/utils.notest.gd

```diff
@@ -254,3 +256,4 @@ static func get_property_usage_string(usage: int) -> String:
 		return "<invalid usage flags>"
 
 	return result.left(-1)
+
```

Comment:
Unnecessary blank line at end of file. There must be exactly one newline (`\n`), not zero or two.